harvard-university-for-the-creative-arts.csl

```diff
@@ -110,11 +125,11 @@
           </else-if>
         </choose>
       </if>
-      <else-if type="manuscript song post-weblog post personal_communication speech" match="any">
-        <group delimiter=". ">
+      <else-if type="manuscript song post-weblog post" match="any">
+        <group delimiter=" ">
           <text variable="genre" prefix="[" suffix="]"/>
           <text variable="medium" prefix="[" suffix="]"/>
-          <text variable="note" prefix="[" suffix="]"/>
+          <text variable="note" suffix="."/>
```

Comment:
Hi

Thanks for getting in touch.

I had the note field there because I am not able to use the Podcast source type to recreate the podcast example from our referencing guide, as there is no date option in Podcast. Our example looks like this:

Conger, C. and Ervin, C. (2015) _How to skate like a girl, part 1_. [audio podcast, online] 29 June 2015. At: http://www.stuffmomnevertoldyou.com/podcasts/how-to-skate-like-a-girlpart-1/ (Accessed on 8 June 2016)

The podcast option doesn't put a year at the start or a date after the medium. So as a way around I was thinking of using Audio Recording as this does have a year at the start of the reference, but with the Extra field for the full date of podcast. I used the Extra field as I don't want the date ending up in other sources which use Audio Recording, e.g. songs which don't require a full date in the reference.

Is there another way to do this? 

Alternatively I can take it out and if you'd rather it wasn't there. 

Ian